play/src/i18n/en-US/mapEditor.ts

```diff
@@ -169,6 +169,16 @@ const mapEditor: BaseTranslation = {
         areaDescriptionPlaceholder: "My area is a...",
         areaSerchable: "Searchable in the exploration mode",
         addDescriptionField: "Add description field",
+        addRight: "Add rights",
+        rightTitle: "Right by user tag (write / read)",
+        rightDescription:
+            "Rights define who can interact with the area. If you leave it empty, everyone can use it. If you set it, only users who have at least one of these 'tags' can use it.",
+        rightWriteTitle: "Write rights",
+        rightWriteDescription:
+            "Write rights define who can modify the area. The user who have at least on of these tags can create, update or delete an object in the area.",
+        rightReadTitle: "Read rights",
```

Comment:
I think we talked about renaming read/write to something else? Did we not?
Something like "edit right" and "enter / use right" ?

Reply:
I don't think so. But I can modify it if you prefer.